Assets/Scripts/WorkPuzzle/FallTimer.cs

```diff
@@ -12,7 +12,7 @@ public class FallTimer : MonoBehaviour
     public WorkManager workManager;
 
     //time between the spawning of each word
-    public static float delay = 1f;
+    public static float delay = 3f;

```

Comment:
How about we make this a public variable we can set for each day? A really easy difficulty increase for each day would be making the work puzzle go faster. We can also play around with reading in input backwards, flipping words etc, but if we don't get there this is a super easy one.